src/wallet/feebumper.cpp

```diff
@@ -243,6 +244,21 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
             ExtractDestination(output.scriptPubKey, change_dest);
             new_coin_control.destChange = change_dest;
         }
+        new_outputs_value += output.nValue;
+    }
+
+    // If no recipients, means that we are sending coins to a change address
+    if (recipients.empty()) {
+        // Just as a sanity check, ensure that the change address exist
+        if (std::get_if<CNoDestination>(&new_coin_control.destChange)) {
+            errors.emplace_back(Untranslated("Unable to create transaction. Transaction must have at least one recipient"));
+            return Result::INVALID_PARAMETER;
+        }
+
+        // Add change as recipient with SFFO flag enabled, so fees are deduced from it.
+        // If the output differs from the original tx output (because the user customized it) a new change output will be created.
```

Comment:
If I'm understanding this correctly, this means that this could result in the creation of a transaction with 2 outputs that our wallet would identify as change. What would happen if we wanted to bump this transaction again, but keep the outputs the same? Though I haven't tested this, I think that the following will happen:

1. The first output is added as `new_coin_control.destChange`
2. Then we find the second output is also change and over-write the original `destChange` with the second output
3. `recipients` is empty, so we make recipient the current `destChange`
4. `destChange` is now nothing, so we have lost the first output address.

I suppose it doesn't really matter, because the money is still going to the same wallet, but usually it is expected for the outputs to be preserved unless `outputs` is used.

Reply:
Yeah, I started working on it here but ended up moving the work to another branch because it's not a problem introduced in this PR.

This PR enables the bump of single output transactions that were previously not possible. It's not changing how we decide which output becomes change.

Anyone can craft a transaction with two outputs sending to change addresses already (just need to call `getrawchangeaddress` twice) and bump it.

That issue is inside the `outputs` loop that is above this code that doesn't check `destChange` existence prior set.
Which well, in the future PR, we will need to decide which change address use if there are many of them.